wesnoth-units.po

```diff
@@ -2005,10 +2039,8 @@ msgstr "louč"
 
 #. [unit_type]: id=Dune Paragon, race=dunefolk
 #: data/core/units/dunefolk/Paragon.cfg:8
-#, fuzzy
-#| msgid "Dragoon"
 msgid "Dune Paragon"
-msgstr "Dragoun"
+msgstr "Přední příslušník pouštního lidu"
```

Comment:
přijde mi to na název jednotky moc dlouhé - jak to vypadá v pravém panelu hry?

zas nevím, co je to za jednotku, ale co třeba Předák pouštního lidu, Pouštní předák...?

Reply:
No to je ta formulace, co jsem hledal!